src/main/msp/msp.c

```diff
@@ -1505,8 +1505,10 @@ static bool mspProcessOutCommand(int16_t cmdMSP, sbuf_t *dst)
         sbufWriteU8(dst, rxConfig()->rc_smoothing_type);
         sbufWriteU8(dst, rxConfig()->rc_smoothing_input_cutoff);
         sbufWriteU8(dst, rxConfig()->rc_smoothing_derivative_cutoff);
-        sbufWriteU8(dst, rxConfig()->rc_smoothing_input_type);
-        sbufWriteU8(dst, rxConfig()->rc_smoothing_derivative_type);
+        // was used for rc_smoothing_input_type, not required in 4.3
+        sbufWriteU8(dst, 0);
```

Comment:
Could we have the comment on the same line as the parameter that was removed?
Usually when we add or remove stuff to MSP we refer to the API version rather than the firmware version. In this case it's removed in MSP API version 1.44.
These types of comments are done in many different ways in this file. I think it would be good if we could settle on a format to use going forward like we already have for adding parameters.
Here's my personal favourite:
https://github.com/betaflight/betaflight/blob/b26e08e6cb86a1de3be9f9dc11f651705d0184cd/src/main/msp/msp.c#L1777-L1779